### PR TITLE
Pdf download

### PR DESCRIPTION
**Pull request by Benet5**

### CODE REVIEW

File: backend/src/main/java/Billprint/Mapping/MappingService.java [hunk 1]
```diff
@@ -77,11 +77,30 @@ public String clientMapping(Item item){
...
+        toChange.get().setSkonto(client.getSkonto());
+        toChange.get().setTax(client.isTax());
+        clientRepo.save(toChange.get());
+    }
+    #
+ */
+
+    public List<ClientToPrint> findAll(){
+        return clientToPrintRepo.findAll();
     }
```
Comment (Droggelbecher92) [inline backend/src/main/java/Billprint/Mapping/MappingService.java:104]:
```suggestion
```
WIP

**Review by Droggelbecher92 — CHANGES_REQUESTED** [review]
Wip

**Review by Droggelbecher92 — APPROVED** [review]
👍

File: backend/src/main/java/Billprint/Mapping/MappingService.java [hunk 2]
```diff
@@ -77,12 +77,28 @@ public String clientMapping(Item item){
...
+            if(clientToPrintRepo.findByAddressName(client.getAddress().getName()).isEmpty()) {
+                ClientToPrint clientToPrint = new ClientToPrint(client, this.importService);
+                ClientToPrint actual = clientToPrintRepo.save(clientToPrint);
+                allConverted.add(actual);
+            }
+
+            }return allConverted;
 
+        }
+/*
```
Comment (gossie) [inline backend/src/main/java/Billprint/Mapping/MappingService.java:89]: If you don't need it, I would delete it.

**Pull request merged**